### PR TITLE
linux: Add 4.11

### PR DESCRIPTION
**Pull request by NeQuissimus**
###### Motivation for this change
4.11 stable kernel

###### Things done

- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by vcunat — APPROVED** [review]
Seems OK, after a brief look.

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -11744,6 +11744,23 @@ with pkgs;
...
   };
 
+  linux_4_11 = callPackage ../os-specific/linux/kernel/linux-4.11.nix {
+    kernelPatches =
+      [ kernelPatches.bridge_stp_helper
+        kernelPatches.p9_fixes
+        # See pkgs/os-specific/linux/kernel/cpu-cgroup-v2-patches/README.md
+        # when adding a new linux version
+        # !!! 4.7 patch doesn't apply, 4.9 patch not up yet, will keep checking
+        # kernelPatches.cpu-cgroup-v2."4.7"
```
Comment (7c6f434c) [inline pkgs/top-level/all-packages.nix]: Maybe it's time to finally update the cpu-cgroup-v2 patches? 4.11 seems up

Comment (NeQuissimus) [inline pkgs/top-level/all-packages.nix]: Ah, yes, let me do that

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -11714,10 +11714,7 @@ with pkgs;
     kernelPatches =
       [ kernelPatches.bridge_stp_helper
         kernelPatches.p9_fixes
-        # See pkgs/os-specific/linux/kernel/cpu-cgroup-v2-patches/README.md
-        # when adding a new linux version
```
Comment (7c6f434c) [inline pkgs/top-level/all-packages.nix]: I think the lines about updating should be kept (looking how we blindly copypasted 4.10 it's better to have some reminder…), but the short-term FIXME below should be deleted

Comment (NeQuissimus) [inline pkgs/top-level/all-packages.nix]: I put it back on the most recent version

**Pull request closed without merging**